FRPs/proposal-privacy.md

```diff
@@ -0,0 +1,34 @@
+---
+id: <>
+title: Cryptographic primitives for complete privacy
+team: @JBStearn
+created: 2021-12-16
+---
+
+# Cryptographic primitives for complete privacy
+
+## Background and Problem Statement
+
+Flashbots’ roadmap sets complete privacy as a core design goal. This means that transactions in the mempool are private until they have been mined on-chain. Complete privacy ensures that no privileged actors (e.g. miners/block proposers) are able to extract MEV through ordering or censorship attacks.  
+
+In this FRP, we will survey the current cryptographic approaches to mempool privacy, including verifiable delay functions, threshold encryption, and multi-party timed commitments. We consider the advantages and disadvantages of these methods in terms of defined properties of a complete privacy solution, such as UX, scalability, etc. We assess the security of these methods with respect to a range of threat models. This section of the FRP will build upon a literature review of the mathematics underlying the respective cryptographic primitives,  
+

```

Comment:
incomplete phrase? was this meant to say that the FRP will include this literature review? or where can we find it?